File: packages/android/app/src/main/java/studio/eugenezakharov/opencode/api/models/CommandInfo.kt
package studio.eugenezakharov.opencode.api.models

import kotlinx.serialization.Serializable

/**
 * A slash command available in the session (`GET /command`) — built-in (e.g.
 * `init`, `review`) or project-defined. Mirrors iOS `CommandInfo`.
 */
@Serializable
data class CommandInfo(
    val name: String,
    val description: String? = null,
)

File: packages/ios/OpenCode/API/Models/CommandInfo.swift
import Foundation

/// A slash command available in the session (`GET /command`) — built-in (e.g.
/// `init`, `review`) or project-defined. Running one posts it as a prompt.
struct CommandInfo: Decodable, Identifiable {
    let name: String
    let description: String?

    var id: String { name }
}
